joomla-projects/gsoc22_seo
# Add Nofollow-Dofollow Link checker

### Summary of Changes

- Add Outgoing Nofollow-Dofollow Link Checker under the SEO tab under ```Article Edit Section```
- The new feature will allow the admin to monitor the external links of the article and categorize them as dofollow and nofollow
- It will enable developers to prevent ‘link juice’ from flowing out to unwanted sites based on the external site's credibility
- Admins can choose to retain or change the ‘rel’ directive (from dofollow to nofollow or vice versa) by going to the 'Content' tab on the same page

![image](https://user-images.githubusercontent.com/84401192/173516703-aa20abc0-a375-4668-8382-41071f76186c.png)


### Testing Instructions

- Go to the 'Content' tab under ```Article Edit Section```
- Click on 'Toggle Editor' and add a few anchor tags to the article
- Change some of the 'rel' attributes of these tags to nofollow
- Now move over to the 'SEO' tab and take a look at the Outgoing Link Checker section
- A list of all the outgoing nofollow and dofollow links will appear


### Actual result BEFORE applying this Pull Request

- The Outgoing Nofollow-Dofollow Link Checker section should not be present under the SEO tab


### Expected result AFTER applying this Pull Request

- The newly added Outgoing Nofollow-Dofollow Link Checker section should be visible
